src/cmd/cmd.cpp

```diff
@@ -533,6 +533,19 @@ int main(int argc, char **argv)
         }
     }
 
+    // Pre-flight check: verify that the file specified by --unsyncedfolders can be read by us:
+    if (!ctx.options.unsyncedfolders.isNull()) { // yes, isNull and not isEmpty because...:
+        // ... if the user entered "--unsyncedfolders ''" on the command-line, opening that will
+        // also fail
+        QFile f(ctx.options.unsyncedfolders);
+        if (!f.open(QFile::ReadOnly)) {
+            qFatal("Cannot read unsyncedfolders file '%s': %s",
+                qPrintable(ctx.options.unsyncedfolders),
```

Comment:
In the gui part I replaced all qPrintable calls with qUtf8Printable, which makes sense as we have a log file that is capable of printing this.
When I did the change I also replaced the qPrintable  in here, probable won't work but we should keep to one?
So maybe change the others back?

Reply:
I changed the other to `qPrintable`.